SETUP.md

```diff
@@ -0,0 +1,101 @@
+# Setup Instructions
+
+To participate in the workshop, you will need access to the software described below. In addition, you will need an up-to-date web browser.
+
+## The Bash Shell
+
+Bash is a commonly-used shell that gives you the power to do simple
+tasks more quickly.
+
+### Windows
+
+<a href="https://www.youtube.com/watch?v=339AEqk9c-8">Video Tutorial</a>
+
+1. Download the Git for Windows <a href="https://git-for-windows.github.io/">installer</a>.
+2. Run the installer and follow the steps bellow:
+  - Click on "Next".
+  - Click on "Next".
+  - **Keep "Use Git from the Windows Command Prompt" selected and click on 
+  "Next".**
+    If you forgot to do this programs that you need for the workshop will not 
+    work properly.
+    If this happens rerun the installer and select the appropriate option.
+  - Click on "Next".
+  - **Keep "Checkout Windows-style, commit Unix-style line endings" selected and 
+  click on "Next".**
+  - ** Keep "Use Windows' default console window" selected and click on "Next".**
+  - Click on "Install".
+  - Click on "Finish".
+3. If your "HOME" environment variable is not set (or you don't know what this is):
+  - Open command prompt (Open Start Menu then type <code>cmd</code> and press [Enter])</li>
+  - Type the following line into the command prompt window exactly as shown:
+    `setx HOME "%USERPROFILE%"`
+  - Press [Enter], you should see `SUCCESS: Specified value was saved.`
+  - Quit command prompt by typing `exit` then pressing [Enter]
+
+This will provide you with Bash in the Git Bash program.
+
+### macOS
+
+The default shell in all versions of macOS is Bash, so no need to install 
+anything. You access Bash from the Terminal (found in `/Applications/Utilities`).
+See the Git installation <a href="https://www.youtube.com/watch?v=9LQhwETCdwY ">
+video tutorial</a> for an example on how to open the Terminal. You may want to 
+keep Terminal in your dock for this workshop.
+
+### Linux
+
+The default shell is usually Bash, but if your machine is set up differently you 
+can run it by opening a terminal and typing `bash`. There is no need to install 
+anything.
+
+## Text Editor
```

Comment:
@darkdrgn2k is there any code writing in your modules? There isn't need for this in mine, and if there is any it would be on the mesh-orange shell, where neither vim or nano is available, and nano-tiny is used. So this section seems unnecessary.

Reply:
Didn't we talk about this last night any everyone said they opened a notepad?

Reply:
Honestly I did not really understand the whole "you need a text editor to copy/paste" thing.  You Copy, You Paste. 

But having a  notepad is a good idea for random reasons.  But i think you will scare every one off talking about VI and git again and whatever.

I dont think this needs to be in the intro email, id take it out.

But i dont deny that 

Again I think this is more confusing then it needs to be. I don't think there is any part of the module where they have to write "code" aside from copying text from the sheet.

Remember these are people who don't sit in console, they work GUI.  So teaching them about VI on day 1 may scare them off!